### PR TITLE
Add fetch_price method in GB parsers based on fetch_price FR parser

### PR DESCRIPTION
**Pull request by AlexandreCouderc**
GB prices are available in the data used by the FR parser to fetch prices for FR.

### CODE REVIEW

**Review by madsnedergaard — APPROVED** [review]
Haha, nice "hack" to use French data for getting GB prices 😄

**Review by madsnedergaard — APPROVED** [review]
Okay, I was a bit too fast here - there's a few chances required before we can merge this :)

1) The parsers you have changed here are for Northern Ireland and the Orkney Islands. We should probably create a new file for this :)
2) The `zones.json` file needs to be updated to use this new parser for `price` :)

**Review by madsnedergaard — CHANGES_REQUESTED** [review]
Okay, I was a bit too fast here - there's a few chances required before we can merge this :)

1) The parsers you have changed here are for Northern Ireland and the Orkney Islands. We should probably create a new file for this :)
2) The `zones.json` file needs to be updated to use this new parser for `price` :)

**Review by madsnedergaard — APPROVED** [review]
Brilliant, thanks for the very quick iteration 🙌

**Comment by madsnedergaard** [comment]
Closes https://github.com/electricitymap/electricitymap-contrib/discussions/3706 and https://github.com/electricitymap/electricitymap-contrib/issues/2922 🥳

File: parsers/GB.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          'teDeb={}&dateFin={}&mode=NORM'.format(formatted_from, formatted_to)
+    response = r.get(url)
+    obj = ET.fromstring(response.content)
+    datas = {}
+
+    for donnesMarche in obj:
+        if donnesMarche.tag != 'donneesMarche':
+            continue
+
+        start_date = arrow.get(arrow.get(donnesMarche.attrib['date']).datetime, 'Europe/London')
```
Comment (AlexandreCouderc) [inline parsers/GB.py:42]: I have a doubt here if I should put 'Europe/London' or 'Europe/Paris' for the timezone.

Comment (madsnedergaard) [reply parsers/GB.py:42]: That is a good question 😄 
Let's see how the data looks now and then we can follow-up on it if it's clear that it's off :)

**Pull request merged**